src/Commands/SecretsCommand.php

```diff
@@ -163,15 +164,16 @@ protected function rsync($site_env_id, $src, $dest)
         $src = preg_replace('/^:/', $siteAddress, $src);
         $dest = preg_replace('/^:/', $siteAddress, $dest);
 
-        $this->passthru("rsync -rlIvz --ipv4 --exclude=.git -e 'ssh -p 2222' $src $dest >/dev/null 2>&1");
+        $this->passthru("rsync -rlIvz --ipv4 --exclude=.git -e 'ssh -p 2222' $src $dest >/dev/null 2>&1",
```

Comment:
I'd rather see the code just call `passthru` here, then handle the error code, rather than perturb the parameters for the passthru convenience wrapper.

Reply:
Picking up the thread of this work - if a goal is to leave the existing parameters for passthru() I can imagine a few options. Do you have a preference?

1. Just ignore all instances of error 23, for both download and upload operations. This seems too heavy-handed IMO.
2. Change the flags passed to rsync to ignore missing-file errors (I'd have to look up what flag that is). This holds some promise, but is also a bit heavy-handed if applied in both directions.
3. Add a check somewhere to see if secrets.json exists before getting this far, and optionally to create it if needed.
4. Add logic inside passthru() to ignore error 23 only for download operations (not sure how I'd do this - the only function I recall in PHP for this is debug_backtrace, which isn't ideal)

Reply:
No, I didn't mean any of that -- I mean just use PHP's `\passthru`, and not use `$this->passthru()` here at all. You'd then need to test for errors that were not 23 on the following line, and `throw` for everything else.